cmd/skywire-visor/commands/root.go

```diff
@@ -334,6 +337,12 @@ func runVisor(conf *visorconfig.V1) {
 			}
 		}
 	}
+	if logLvl != "" {
+		if logLvl == "INFO" || logLvl == "DEBUG" || logLvl == "TRACE" {
+			log.Info("setting log level to: ", logLvl)
+			conf.LogLevel = logLvl
+		}
+	}
```

Comment:
Be good if the behave of this condition not limited to capital words, means set `info` and `Info` works like `INFO`. Currently just `INFO` and `TRACE` and `DEBUG` work well.